app/hooks/queries/useHistory.ts

```diff
@@ -0,0 +1,50 @@
+import { useQuery } from '@tanstack/react-query';
+import { Instance } from 'app/api';
+
+export interface ProductHistory {
+  result: [
+    {
+      productName: string;
+      productImage: string;
+      price: number;
+      size: number;
+      createdAt: string;
+      deadLine: null;
+      tradedAt: string;
+      type: string;
+      status: string;
+    }
+  ];
+}
+
+interface ProductHistoryProps {
+  type: string;
+  status: 'bidding' | 'pending' | 'finished';
+  startDate: string;
+  endDate: string;
+}
+
+const useProductHistory = ({
+  type,
+  status,
+  startDate,
+  endDate,
+}: ProductHistoryProps) => {
+  return useQuery<ProductHistory>({
+    queryKey: ['type', type, status],
```

Comment:
```suggestion
    queryKey: ['product-history', type, status],
```
queryKey는 이름과 비슷하게 쓰는걸 추천드립니다..!